packages/ai-code-completion/src/browser/ai-code-completion-preference.ts

```diff
@@ -39,6 +40,14 @@ export const AICodeCompletionPreferencesSchema: PreferenceSchema = {
                 type: 'string'
             },
             default: []
+        },
+        [PREF_AI_INLINE_COMPLETION_MAX_CONTEXT_LINES]: {
+            title: 'Maximum Context Lines',
+            type: 'number',
+            description: 'The maximum number of lines shared between the prefix and suffix for AI code completion.\
```

Comment:
Are the terms `prefix` and `suffix` something the user can understand? These are agent-specific variables that may not be obvious to the user so I wonder whether we shouldn't replace that with something like:
`The maximum number of lines distributed among the lines before and after the cursor position.`

Reply:
Great point, adapted description!